agent.py

```diff
@@ -14,18 +14,24 @@
 initialize_logging('collector')
 
 # stdlib
+from copy import copy
 import logging
 import os
 import signal
 import sys
 import time
-import supervisor.xmlrpc
 import xmlrpclib
-from copy import copy
 
 # For pickle & PID files, see issue 293
 os.umask(022)
 
+# 3p
+try:
+    import supervisor.xmlrpc
+except ImportError:
+    # Not used/shipped on Windows
+    pass
```

Comment:
I respectfully disagree. I don't think it's needed for one dependency. Actually I would prefer extracting the function using `supervisor.xmlrpc` in its own `utils/` and have the logic there.